inlong-sort/sort-connectors/doris/src/main/java/org/apache/inlong/sort/doris/table/DorisDynamicSchemaOutputFormat.java

```diff
@@ -486,6 +497,17 @@ private void handleDirtyData(Object dirtyData, DirtyType dirtyType, Exception e)
             }
             throw ex;
         }
+
+        if (multipleSink) {
+            String[] tableWithDb = tableIdentifier.split("\\.");
```

Comment:
It is not right because of the tableIdentifer is dynamic when it it multiplesink.

Reply:
the table identifier.split is used for multiple sink regular out data count, so I think it works for dirty data too...? or is there something i missed